src/main/resources/views/finish.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="utf-8"/>
+    <link rel="stylesheet" type="text/css"
+          th:href="@{/resources/css/style.css}"/>
+</head>
+<body>
+<div class="container">
+    <header th:replace="header :: header"></header>
+    <form class="form-horizontal"
+          th:action="@{'/courses/'+${course.getId()}+'/finish'}" method="POST">
+        <fieldset>
+            <div class="control-group">
+                <div class="control-label">
+                    Course
+                </div>
+                <div id="titleField" class="controls text"
+                     th:text="${course.getCourseName()}">
```

Comment:
i think this can be rewritten in using java bean convention

${course.courseName}

This is more readable.